index.js

```diff
@@ -2,4 +2,8 @@
 
 module.exports = {
   name: require('./package').name,
+
+  options: {
+    'ember-cli-babel': { enableTypeScriptTransform: true },
```

Comment:
Is this really needed? On my mind `index.js` file was used by consuming project and it looks strange to me to have a typescript setting here

Reply:
https://github.com/ember-cli/ember-cli/blob/v5.4.0/blueprints/addon/files/index.js